src/NServiceBus.Core/MessagingBestPractices/BestPracticesEnforcement.cs

```diff
@@ -0,0 +1,29 @@
+namespace NServiceBus.Features
+{
+    using NServiceBus.MessagingBestPractices;
+    using NServiceBus.Pipeline;
+
+    /// <summary>
+    /// Makes sure that messaging best practices are followed
+    /// </summary>
+    public class BestPracticesEnforcement : Feature
```

Comment:
You [pick on my class naming](https://github.com/Particular/NServiceBus/pull/2689#discussion_r29410570), and now you create a pluralized class :wink: